src/main/java/org/broadinstitute/hellbender/engine/dataflow/transforms/GoogleGenomicsReadToGATKRead.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.engine.dataflow.transforms;
+
+import com.google.api.services.genomics.model.Read;
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import org.broadinstitute.hellbender.utils.read.GoogleGenomicsReadToGATKReadAdapter;
+import org.broadinstitute.hellbender.utils.read.MutableGATKRead;
+
+
+public class GoogleGenomicsReadToGATKRead extends PTransform<PCollection<Read>, PCollection<MutableGATKRead>> {
```

Comment:
Having transforms as separate classes encourages good testing/documentation practices, and discourages duplicated nested classes that do the same thing.